File: TableTest/TableTest/ViewController.m
//
//  ViewController.m
//  TableTest
//
//  Created by Ipageon_김태형 on 2022/10/07.
//

#import "ViewController.h"
#import <NotificationCenter/NotificationCenter.h>
#import "SecondViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UITextField *tFiled;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.tFiled.delegate = self;
}

- (void)textFieldDidEndEditing:(UITextField *)textField{


    
}

- (IBAction)onClickButton:(id)sender {
//    UIViewController *uv = [[UIViewController alloc] initWithNibName:@"SecondViewController" bundle:NSBundle.mainBundle];
    
//    SecondViewController* uv =[self.storyboard instantiateViewControllerWithIdentifier:@"SecondViewController"];
//    svc.anotherString = self.originalString;
//    [self presentViewController:svc animated:YES completion:nil];

    
//    [self presentViewController:uv animated:YES completion:nil];
    
    
}
- (IBAction)qqqq:(id)sender {
//    SecondViewController* uv =[self.storyboard instantiateViewControllerWithIdentifier:@"SecondViewController"];
//    svc.anotherString = self.originalString;
//    [self presentViewController:svc animated:YES completion:nil];
    
        NSString *name = self.tFiled.text;

        [NSNotificationCenter.defaultCenter postNotificationName:@"name"
                                                          object:nil
                                                        userInfo:@{
            @"name" : name
        }];
    
//    [self presentViewController:uv animated:YES completion:nil];
    
}

@end

File: SampleProjectTHKim/SampleProjectTHKim/ViewController.swift
//
//  ViewController.swift
//  SampleProjectTHKim
//
//  Created by Ipageon_김태형 on 2021/12/28.
//

import UIKit

class ViewController: UIViewController {
    @IBOutlet weak var inputEmail: UITextField!
    
    @IBOutlet weak var inputPassword: UITextField!
    @IBOutlet weak var loginButton: UIButton!
    
    @IBOutlet weak var goToSignup: UIButton!
    
    override func viewDidLoad() {
        super.viewDidLoad()
    }
    
    override func viewWillAppear(_ animated: Bool) {
        goToSignup.layer.cornerRadius = 5
        loginButton.layer.cornerRadius = 5
    }
    
    
    @IBAction func onClickLogin(_ sender: Any) {
    }
    
    @IBAction func onClickSignup(_ sender: Any) {
    }
    

}

